lib/manageiq_performance/middlewares/active_support_timers.rb

```diff
@@ -19,6 +19,14 @@ def active_support_timers_initialize
       def active_support_timers_start(env); end
 
       def active_support_timers_finish(env)
+        if Thread.current[:miq_perf_request_timer_data].nil? and env["HTTP_MIQ_PERF_TIMESTAMP"]
+          time = (env["MIQ_PERFORMANCE_END_TIME"] - env["HTTP_MIQ_PERF_TIMESTAMP"]) / 1000
```

Comment:
Not opposed to the idea, but do you have a reason one way or the other for why we should change it?

I think when it was coming in from the `Requestor`, it was passed in as a string only because it is a HTTP header, versus when I did it in the `.profile` method, it was an int (and `env` at that point was a fake env hash at that point).

Again, don't have a strong opinion, just curious if there is a strong rational for moving to strings.